docker/docker.engine.yaml

```diff
@@ -0,0 +1,185 @@
+    password: ${env:REDIS_PASSWORD}
+
+auth:
+  token_store:
+    redis:
+      addr: ${env:REDIS_HOST}:${env:REDIS_PORT}
+      password: ${env:REDIS_PASSWORD}
+
+  providers:
+    - id: default-atlassian
+      description: "The default Atlassian provider"
+      enabled: false
+      type: oauth2
+      provider_id: atlassian
+      client_id: ${env:ATLASSIAN_CLIENT_ID}
+      client_secret: ${env:ATLASSIAN_CLIENT_SECRET}
+
+    - id: default-discord
+      description: "The default Discord provider"
+      enabled: false
+      type: oauth2
+      provider_id: discord
+      client_id: ${env:DISCORD_CLIENT_ID}
+      client_secret: ${env:DISCORD_CLIENT_SECRET}
+
+    - id: default-dropbox
+      description: "The default Dropbox provider"
+      enabled: false
+      type: oauth2
+      provider_id: dropbox
+      client_id: ${env:DROPBOX_CLIENT_ID}
+      client_secret: ${env:DROPBOX_CLIENT_SECRET}
+
+    - id: default-github
+      description: "The default GitHub provider"
+      enabled: false
+      type: oauth2
+      provider_id: github
+      client_id: ${env:GITHUB_CLIENT_ID}
+      client_secret: ${env:GITHUB_CLIENT_SECRET}
+
+    - id: default-google
+      description: "The default Google provider"
+      enabled: false
+      type: oauth2
+      provider_id: google
+      client_id: ${env:GOOGLE_CLIENT_ID}
+      client_secret: ${env:GOOGLE_CLIENT_SECRET}
+
+    - id: default-linkedin
+      description: "The default LinkedIn provider"
+      enabled: false
+      type: oauth2
+      provider_id: linkedin
+      client_id: ${env:LINKEDIN_CLIENT_ID}
+      client_secret: ${env:LINKEDIN_CLIENT_SECRET}
+
+    - id: default-microsoft
+      description: "The default Microsoft provider"
+      enabled: false
+      type: oauth2
+      provider_id: microsoft
+      client_id: ${env:MICROSOFT_CLIENT_ID}
+      client_secret: ${env:MICROSOFT_CLIENT_SECRET}
+
+    - id: default-slack
+      description: "The default Slack provider"
+      enabled: false
+      type: oauth2
+      provider_id: slack
+      client_id: ${env:SLACK_CLIENT_ID}
+      client_secret: ${env:SLACK_CLIENT_SECRET}
+
+    - id: default-spotify
+      description: "The default Spotify provider"
+      enabled: false
+      type: oauth2
+      provider_id: spotify
+      client_id: ${env:SPOTIFY_CLIENT_ID}
+      client_secret: ${env:SPOTIFY_CLIENT_SECRET}
+
+    - id: default-x
+      description: "The default X provider"
+      enabled: false
+      type: oauth2
+      provider_id: x
+      client_id: ${env:X_CLIENT_ID}
+      client_secret: ${env:X_CLIENT_SECRET}
+
+    - id: default-zoom
+      description: "The default Zoom provider"
+      enabled: false
+      type: oauth2
+      provider_id: zoom
+      client_id: ${env:ZOOM_CLIENT_ID}
+      client_secret: ${env:ZOOM_CLIENT_SECRET}
+
+cloud:
+  coordinator_url: ${env:ARCADE_COORDINATOR_URL}
+  identity_url: ${env:ARCADE_IDENTITY_URL}
+
+dashboard:
+  client_id: ${env:ARCADE_DASHBOARD_CLIENT_ID}
```

Comment:
```suggestion

```

Reply:
Remove `cloud` and `dashboard` (internal/hidden options). The Engine defaults are correct without these values set in `engine.yaml`.